lib/hasPermissions.ts

```diff
@@ -200,20 +206,51 @@ export class HasPermissionsDirectiveVisitor<
       location === DirectiveLocation.INPUT_OBJECT ||
       location === DirectiveLocation.ARGUMENT_DEFINITION;
 
+    const getDefaultValue = (
+      container: GraphQLArgument | GraphQLInputObjectType | GraphQLObjectType,
+      path?: Array<string>,
+    ): unknown => {
+      if ('defaultValue' in container) {
+        return container.defaultValue;
+      }
+      /**
+       * Used this, because the container is verified before and just can be Argument or InputObjectType
+       * and the path in a InputObjectType just can be undefined when the input doesn't have variables inside
+       * but this is don't accepted for sintax GraphQL, so is very difficult test this, therefore, the last
+       * case of the function is very difficult to test too, when the return is undefined;
```

Comment:
nit:
```suggestion
       * Used this because the container is verified before and just can only be `Argument` or `InputObjectType`.
       * The path in a `InputObjectType` can only be undefined when the input doesn't have any fields,
       * which is not accepted in GraphQL syntax.
       * due to the aforementioned reasons, it is difficult to test this case(if not impossible)
```